Dockerfile

```diff
@@ -90,6 +90,7 @@ RUN pip3 install --pre torchvision --extra-index-url https://download.pytorch.or
                     pytest-cov pytest-xdist pytest-sugar pytest-html \
                     lightning \
     && pip install -U Pillow
```

Comment:
```suggestion
    && pip install -U Pillow \
```

Reply:
Yes, must've missed this character when making the commit.

Tested locally, here is the traceback before:
```
Traceback (most recent call last):
  File "/workspace/machop/mase_components/resblock/test/timing_test_resblock.py", line 21, in <module>
    from chop.passes.graph.transforms.quantize.quantized_modules import Conv2dInteger, LinearInteger, ReLUInteger
  File "/workspace/machop/chop/__init__.py", line 1, in <module>
    from .tools.logger import root_logger
  File "/workspace/machop/chop/tools/__init__.py", line 11, in <module>
    from .get_input import get_cf_args, get_dummy_input
  File "/workspace/machop/chop/tools/get_input.py", line 3, in <module>
    from ..models.utils import ModelSource
  File "/workspace/machop/chop/models/__init__.py", line 6, in <module>
    from .patched import (
  File "/workspace/machop/chop/models/patched/__init__.py", line 2, in <module>
    from .opt_patched import (
  File "/workspace/machop/chop/models/patched/opt_patched/__init__.py", line 3, in <module>
    from .modeling_opt_patched import (
  File "/workspace/machop/chop/models/patched/opt_patched/modeling_opt_patched.py", line 19, in <module>
    import torch.utils.checkpoint
  File "/usr/local/lib/python3.11/dist-packages/torch/utils/checkpoint.py", line 23, in <module>
    from torch._functorch._aot_autograd.functional_utils import is_fun
  File "/usr/local/lib/python3.11/dist-packages/torch/_functorch/_aot_autograd/functional_utils.py", line 13, in <module>
    from torch.fx.experimental.symbolic_shapes import definitely_true, sym_eq
  File "/usr/local/lib/python3.11/dist-packages/torch/fx/experimental/symbolic_shapes.py", line 52, in <module>
    from torch.utils._sympy.functions import FloorDiv, Mod, IsNonOverlappingAndDenseIndicator
  File "/usr/local/lib/python3.11/dist-packages/torch/utils/_sympy/functions.py", line 1, in <module>
    import sympy
  File "/usr/local/lib/python3.11/dist-packages/sympy/__init__.py", line 30, in <module>
    from sympy.core.cache import lazy_function
  File "/usr/local/lib/python3.11/dist-packages/sympy/core/__init__.py", line 9, in <module>
    from .expr import Expr, AtomicExpr, UnevaluatedExpr
  File "/usr/local/lib/python3.11/dist-packages/sympy/core/expr.py", line 4159, in <module>
    from .mul import Mul
  File "/usr/local/lib/python3.11/dist-packages/sympy/core/mul.py", line 2193, in <module>
    from .numbers import Rational
  File "/usr/local/lib/python3.11/dist-packages/sympy/core/numbers.py", line 4567, in <module>
    _sympy_converter[type(mpmath.rational.mpq(1, 2))] = sympify_mpmath_mpq
                          ^^^^^^^^^^^^^^^
AttributeError: module 'mpmath' has no attribute 'rational'
```
Afterwards this is gone. This happens with mpmath version 1.4.0